pkg/visibility/api/conversion_test.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package v1alpha1_test
+package api_test
```

Comment:
I have a better solution:

Keep the file where it was, then:

```suggestion
package v1alpha1
```

And just instantiate a new scheme:

```golang
scheme := runtime.NewScheme()
AddToScheme(scheme)
codec := runtime.NewParameterCodec(scheme)
```

Reply:
Ok. This was the original idea.  Please take a look.